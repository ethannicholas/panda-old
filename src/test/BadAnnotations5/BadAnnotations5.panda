class BadAnnotations5 {
    @limited
    method broken1() {
    }

    @limited
    @class
    method broken2(s:String, i:Int) {
    }
}