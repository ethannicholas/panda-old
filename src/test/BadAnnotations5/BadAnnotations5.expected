error: BadAnnotations5.panda:3:5: '@limited' may not be used on methods which do not take any mutable parameters; did you mean to use '@self' instead?
error: BadAnnotations5.panda:8:5: '@limited' may not be used on methods which do not take any mutable parameters; did you mean to use '@self' instead?
2 errors
